scripts/jest/test-framework-setup.js

```diff
@@ -58,3 +58,26 @@ env.afterEach(() => {
   expect(console.error).toBeReset();
   expect(console.error).toNotHaveBeenCalled();
 });
+
+function wrapDevMatcher(obj, name) {
+  const original = obj[name];
+  obj[name] = function devMatcher() {
+    try {
+      original.apply(this, arguments);
+    } catch (e) {
```

Comment:
It used to be the case that `expect`s didn't throw. Instead it just marked the test as failed and continued on. Has this changed with newer Jasmine versions? Does expecting a throw always work?